### ISSUE TITLE
How many issues and comments are posted on the Exercism organisation?

### ISSUE THREAD

**Issue by iHiD** [exercism/open-source-stats#2]
Issue #1 tracks how many commits are on Exercism, but an equally important metric is how much conversation is going on. I would love some answers and a script, written in Ruby, using the GitHub API, to the question of how many issues and comments are posted across the whole Exercism organisation, preferably broken down into repos, but also with totals.

**Comment by sshine** [comment]
And Ruby because it can be refactored into a website plugin afterwards?

**Comment by iHiD** [comment]
Correct. I'd accept a PR in anything and someone can rewrite later, but it would be nice if all the scripts were consistent.
